temporal-envconfig/build.gradle

```diff
@@ -0,0 +1,13 @@
+description = '''Temporal Java SDK Environment Config Module'''
```

Comment:
In terms of naming it would probably more consistent internally in the Java SDK to call this module `temporal-enviorment-configuration`, but that isn't consistent with the Go SDK

Reply:
:+1: The `envconfig` shortened term for the modules/packages/namespaces myself is kinda nice